tools/android/packaging/Makefile.in

```diff
@@ -102,7 +102,7 @@ python: | xbmc/assets
 	cd xbmc/assets/python2.7/lib/python2.7/; rm -rf test config lib-dynload
 
 res:
-	mkdir -p xbmc/res xbmc/res/raw xbmc/res/values images
+	mkdir -p xbmc/gen/org/xbmc/kodi xbmc/res xbmc/res/raw xbmc/res/values images
```

Comment:
i'd agree with koying. The @APP_NAME_LC@ needs to be revised in general as it doesn't replace the org.xbmc and we don't really want others using that either. So lets solve that in another PR